src/foam/java/refinements.js

```diff
@@ -384,9 +384,15 @@ foam.CLASS({
     function createJavaPropertyInfo_(cls) {
       var info = this.SUPER(cls);
       var m = info.getMethod('cast');
-      m.body = 'return ( o instanceof Number ) ?'
-            + '((Number)o).intValue() :'
-            + '(int)o;';
+      m.body =
+      `if ( o instanceof Number ) {
```

Comment:
Why completely change the formatting rather than just adding an extra clause to the existing condition?